doc/tutorial/Authentication.lhs

```diff
@@ -281,17 +281,22 @@ lookupAccount key = case Map.lookup key database of
 For generalized authentication, servant exposes the `AuthHandler` type,
 which is used to wrap the `Request -> Handler user` logic. Let's
 create a value of type `AuthHandler Request Account` using the above `lookupAccount`
-method:
+method (note: we depend upon `cookie`'s `parseCookies` for this):
```

Comment:
Not annoying, I agree it would be good.

Is there a way to link it easily? Some markdown magic I don't know? Perhaps it's just `[text](http://link)`?